WordPress/src/main/res/layout/reader_site_header_view.xml

```diff
@@ -1,113 +1,94 @@
+                tools:text="Pain in The Bud" />
 
-            <View
+            <com.google.android.material.textview.MaterialTextView
+                android:id="@+id/text_domain"
+                style="@style/ReaderTextView.Site.Header.Caption"
                 android:layout_width="match_parent"
-                android:layout_height="1dp"
-                android:layout_gravity="center_vertical"
-                android:layout_weight="1"
-                android:background="?android:attr/listDivider" />
+                android:layout_height="wrap_content"
+                android:layout_below="@+id/text_blog_name"
+                android:layout_toEndOf="@+id/image_blavatar"
+                android:paddingBottom="@dimen/margin_extra_large"
+                android:paddingTop="@dimen/margin_none"
+                tools:text="www.sitedomainurl.com" />
 
-            <org.wordpress.android.widgets.WPTextView
+            <com.google.android.material.textview.MaterialTextView
                 android:id="@+id/text_blog_follow_count"
+                style="@style/ReaderTextView.Site.Header.Caption"
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
-                android:textAppearance="?attr/textAppearanceCaption"
-                tools:text="12 followers" />
+                android:layout_below="@+id/follow_button"
+                android:layout_toEndOf="@+id/image_blavatar"
+                tools:text="52 followers" />
 
-            <View
-                android:layout_width="match_parent"
-                android:layout_height="1dp"
-                android:layout_gravity="center_vertical"
-                android:layout_weight="1"
-                android:background="?android:attr/listDivider" />
-        </LinearLayout>
+            <org.wordpress.android.ui.reader.views.ReaderFollowButton
+                android:id="@+id/follow_button"
+                style="@style/Reader.Follow.Button.New"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_below="@+id/text_domain"
+                android:layout_toEndOf="@+id/image_blavatar"
+                android:visibility="gone"
+                tools:visibility="visible" />
+        </RelativeLayout>
 
-    </LinearLayout>
+        <com.google.android.material.textview.MaterialTextView
+            android:id="@+id/text_blog_description"
+            style="@style/ReaderTextView.Site.Header.Body"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:ellipsize="end"
+            android:gravity="start"
+            android:maxLines="3"
+            android:paddingEnd="@dimen/margin_extra_large"
+            android:paddingStart="@dimen/margin_extra_large"
+            android:paddingTop="@dimen/margin_medium"
+            android:paddingBottom="@dimen/margin_extra_large"
```

Comment:
Can these be moved to `ReaderTextView.Site.Header.Body` style?

Reply:
> Can these be moved to `ReaderTextView.Site.Header.Body` style?

Sure can, see commit 6338b46